docs/build/wasm/transaction-fees.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 2
+---
+
+# Transaction Fees
+
+## Weight
+
+As is also the case with Substrate, `pallet-contracts` uses [weightV2][weight] to charge execution fees. It is composed of `refTime` and `proofSize` :
+- refTime: The amount of computational time that can be used for execution, in picoseconds.
+- proofSize: The amount of storage space that can be used (also called storage bandwidth), in bytes. So access storage assume that it will grow the gas fees.
```

Comment:
> he amount of storage space that can be used (also called storage bandwidth)
This is misleading - "the amount of storage space than can be used".
You could have a smart contract create 100 MB worth of storage values, while having the PoV size negligible small (e.g. 1-2 kB).

It's the size of data that needs to be included in the proof of validity to verify transaction's state change by the relay chain.

> So access storage assume that it will grow the gas fees.
Think there's a grammatical typo here, not sure what this means :slightly_smiling_face: 
But keep in mind we only charge for the `refTime` now, not the PoV size. Hope this changes soon (seems like a trivial change, but maybe I'm not seeing some problem with it).